prediction/src/tests/test_endpoints.py

```diff
@@ -4,13 +4,13 @@
 
     Provides unit tests for the API endpoints.
 """
+from flask import url_for
 from functools import partial
 import json
-
 import pytest
-
-from flask import url_for
-from src.factory import create_app
+import sys
+sys.path.append('.')
+from src.factory import create_app  # NOQA
```

Comment:
Very good idea, thanks @isms ! [That](https://github.com/concept-to-clinic/concept-to-clinic/pull/61/files#diff-68a7f9edd4ffbca6adb99e2c165cfd52R31) should implement it this way